test/check-application

```diff
@@ -906,9 +906,12 @@ class TestFiles(testlib.MachineCase):
         # Copy/paste file
         m.execute("""
             runuser -u admin mkdir /home/admin/newdir
-            runuser -u admin echo "test_text" > /home/admin/newfile
+            runuser -u admin touch /home/admin/newfile
+            echo "test_text" > /home/admin/newfile
```

Comment:
What does that change? If the file already exists, then echoing as root should not change the permissions. And if you do expect that, why touch it first?

Reply:
Good point, I should just echo as runuser...